src/__test__/actions/assetMakes.actions.test.js

```diff
@@ -52,7 +52,8 @@ describe('Asset Makes action tests', () => {
   it('should dispatch DROPDOWN_ASSET_MAKES_SUCCESS when loadAssetMakesDropdown called successfully', () => {
     mock.onGet().reply(200, mockAssetMakes);
     return store.dispatch(loadAssetMakesDropdown()).then(() => {
-      expect(store.getActions()[0].type).toEqual(DROPDOWN_ASSET_MAKES_SUCCESS);
+      expect(store.getActions()[0].type).toEqual(LOADING_ASSET_MAKES);
```

Comment:
You can use `toContainEqual` for testing, thereby eliminating the need for tracking the action index. Check out [this test](https://github.com/sachungo/movies/blob/master/resources/assets/js/actions/__tests__/actions.test.js) I recently wrote in one of my practice projects.

Also, ensure you follow the `Single Responsibility` priniciple when writing tests. This means that this test suite should test for `DROPDOWN_ASSET_MAKES_SUCCESS` only. The test for `LOADING_ASSET_MAKES` should be a different one